Moderation-Policy.md

```diff
@@ -0,0 +1,116 @@
+* If the author of the Post disagrees that Moderation is required, the matter can be escalated to the TSC by creating a new issue (or using an existing one) within the nodejs/moderation repository with the label `tsc-agenda`. In such cases, the TSC will serve as the final arbiter.
+* While Moderation of a Collaborator's Post is in dispute, no Moderation action should be taken until a decision by the TSC is made.
+* In extreme circumstances involving either obvious gross violations of the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md) or possible compromise of a Collaborator's GitHub account, the TSC can be consulted to waive the 24 hour grace period and dispute process.
+* For Moderation issues involving a TSC member, the member in question is expected to recuse themselves from any decisions required to resolve the issue.
+* When Moderating any Post authored by another Collaborator, the moderating Collaborator must:
+ * Explain the justification for Moderating the post,
+ * Identify all changes made to the Post, and
+ * Identify the steps previously taken to resolve the issue.
+* Explanations of Moderation actions on Collaborator Posts must be provided in:
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Any Collaborator who habitually authors Posts that must be Moderated will be subject to possible removal from the Node.js GitHub Organization. Such action can only be taken through normal TSC motion and vote.
+
+#### Non-Collaborator Posts
+
+* Posts authored by non-Collaborators are always subject to immediate Moderation by any Collaborator if the content is intentionally disruptive or in violation of the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md).
+* When Moderating non-Collaborator Posts, the moderating Collaborator should:
+ * Explain the justification for Moderating the post, and
+ * Identify all changes made to the Post.
+* If an explanation of a Moderation action for a non-Collaborator Post is provided, it must be provided in:
+ * The original Post being modified (as replacement or appended content),
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Moderation of Posts authored by non-Collaborators may result in those non-Collaborators being Banned from further participation in the Node.js GitHub Organization for an indefinite period of time.
+
+Note that Moderating non-Collaborator posts can often lead to retaliation or escalation of inappropriate behavior by the individual whose post is being Moderated. This is true primarily of individuals whose intent is to harass, disrupt or annoy individual members of the community. In such cases, it is best to handle the Moderation as quickly and as quietly as possible without drawing any further undue attention to the Post in question.
```

Comment:
I suggest we change offensive and/or troll issue titles to `-`, since it gives those issues even less visibility in searches.

We should also change the content to something similar.